SonarQube.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -413,36 +413,54 @@
          
        We want to run after the core "ResolveCodeAnalysisRuleSet" target which
        assigns a value to "ResolvedCodeAnalysisRuleSet".
-  -->
+       
+       Conditions:
+       * project is a test project -> turn off Roslyn analysis
+       * project is excluded -> turn off Roslyn analysis
+       * downloaded SonarLint ruleset and analyzer found -> add SonarLint config to existing analysis config
+       * downloaded SonarLint ruleset and analyzer not found -> no change to existing config
+   -->
```

Comment:
As previously discussed, we now want to preserve any existing analyzer settings to enable the user to manually configure additional analyzers. Consequently, we don't automatically override/disable existing analyzers settings.